CHANGELOG.md

```diff
@@ -29,7 +39,7 @@ All notable changes to this project will be documented in this file.
 
 ### Updated
 
-- `operator-rs` `0.56.1` -> `0.57.0` ([#494]).
+- `operator-rs` `0.56.1` -> `0.57.0` ([#494]).z
```

Comment:
fat finger :) ?

Reply:
Are you calling me fat?
[`8547ecd` (#583)](https://github.com/stackabletech/druid-operator/pull/583/commits/8547ecdd3987df262d19d14eeac10b656efe9e20)

Reply:
Not you of course, your fingers!